src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -91,30 +109,36 @@
 
     @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
-            doc="File to which results should be written (defaults to stdout)")
+            doc="File to which the tab-delimited table is written (defaults to stdout)")
     private String out = null;
 
     /**
      * -F NAME can be any standard VCF column (CHROM, ID, QUAL) or any annotation name in the INFO field (e.g., -F AC).
      * To capture GENOTYPE (FORMAT) field values, see the -GF argument. This argument accepts any number
      * of inputs.  So -F CHROM -F POS is allowed.
```

Comment:
-> This argument accepts any number of values, e.g. `-F CHROM -F POS`.

Reply:
done

Reply:
also done for -GF